deepchem/feat/materials_featurizers.py

```diff
@@ -0,0 +1,178 @@
+"""
+Featurizers for inorganic crystals.
+"""
+
+import numpy as np
+
+from deepchem.feat import Featurizer
+from deepchem.utils import pad_array
+
+from matminer.featurizers.composition import ElementProperty
+
+from pymatgen import Composition, Structure
+from pymatgen.analysis.graphs import StructureGraph
+from pymatgen.analysis.local_env import MinimumDistanceNN
+
+
+class ChemicalFingerprint(Featurizer):
+  """
+  Chemical fingerprint of elemental properties from composition.
+
+  Parameters
+  ----------
+  data_source : str, optional (default "matminer")
+      Source for element property data ("matminer", "magpie", "deml")
+  """
+
+  def __init__(self, data_source='matminer'):
+    self.data_source = data_source
+
+  def _featurize(self, comp):
+    """
+    Calculate chemical fingerprint from crystal composition.
+
+    Parameters
+    ----------
+    comp : Reduced formula of crystal.
+    """
+
+    # Get pymatgen Composition object
+    c = Composition(comp)
+
+    ep = ElementProperty.from_preset(self.data_source)
+
+    try:
+      feats = ep.featurize(c)
+    except:
+      feats = []
+
+    return feats
+
+
+class SineCoulombMatrix(Featurizer):
+  """
+  Calculate sine Coulomb matrix for crystals.
+
+  Variant of Coulomb matrix for periodic crystals
```

Comment:
Can you expand out this docstring as mentioned above? Usually a good paragraph or two is the right length for documentation for featurizers